src/main/java/chess/controller/WebChessController.java

```diff
@@ -29,45 +29,57 @@ public WebChessController(final ChessService chessService) {
     }
 
     @PostMapping
-    public ResponseEntity<?> createRoom(@RequestBody RoomRequestDto roomRequestDto) {
+    public ResponseEntity<RoomResponseDto> createRoom(@RequestBody RoomRequestDto roomRequestDto) {
         final RoomResponseDto room = chessService.createRoom(roomRequestDto);
-        return ResponseEntity.created(URI.create("/api/chess/rooms/" + room.getId())).build();
+        return ResponseEntity.status(HttpStatus.CREATED)
+            .location(URI.create("/api/chess/rooms/" + room.getId()))
+            .body(room);
     }
 
     @GetMapping
     public ResponseEntity<RoomsResponseDto> findRooms() {
-        return ResponseEntity.ok(chessService.findRooms());
-    }
-
-    @GetMapping("/{id}/enter")
-    public ResponseEntity<Object> enterRoom(@PathVariable Long id) {
-        final RoomEntity roomEntity = chessService.enterRoom(id);
-        if (roomEntity == null) {
-            return ResponseEntity.notFound().build();
-        }
-        return ResponseEntity.ok().build();
+        return ResponseEntity.status(HttpStatus.OK)
+            .body(chessService.findRooms());
     }
 
     @GetMapping("/{id}")
-    public ResponseEntity<GameResponseDto> getCurrentBoard(@PathVariable Long id) {
-        return ResponseEntity.ok(chessService.getCurrentBoard(id));
+    public ResponseEntity<GameResponseDto> enterRoom(@PathVariable Long id) {
+        return ResponseEntity.status(HttpStatus.OK)
+            .body(chessService.getCurrentBoards(id));
     }
 
-    @PostMapping("/{id}/move")
-    public ResponseEntity<GameResponseDto> movePiece(@PathVariable Long id,
-                                                     @RequestBody MoveRequestDto moveRequestDto) {
-        return ResponseEntity.ok(chessService.move(id, moveRequestDto));
+    @GetMapping("/{id}/restart")
+    public ResponseEntity<GameResponseDto> reEnterRoom(@PathVariable Long id) {
+        chessService.reCreateRoom(id);
```

Comment:
저는 reenter, recreate 라는 단어가 있기에 아래와 같이 사용하는 게 익숙하네요..! 제가 영어가 짧기에 모르는 부분이 있다면 설명해주세용. 제 학습에도 도움될거 같아요 😀

```suggestion
    public ResponseEntity<GameResponseDto> reenterRoom(@PathVariable Long id) {
        chessService.recreateRoom(id);
```

Reply:
제가 더 짧은 것 같습니다! restart를 쓴다면, 토니 말씀대로 똑같이 recreate로 가는 것이 통일성을 유지하는 것 같습니다! 감사합니다 토니!